pyfstat/mcmc_based_searches.py

```diff
@@ -1033,44 +1052,57 @@ def _add_prior_to_corner(self, axes, samples, add_prior):
             for k in range(0, i):
                 axes[i][k].set_ylim(xlim[0], xlim[1])
 
+    def _get_prior_bounds(self, normal_stds=2):
```

Comment:
I wonder if we should be more agressive and set normal_stds=3.

Otherwise, I like it.

Reply:
That was taken from the original `plot_prior_posterior()` where I stole most of this per-prior code. But since that one has its own default which can be different, and here we really want to be reasonably sure we are conservative, it's a good idea to go higher by default.

Reply:
I've added another commit that changes the default to 3 (at the `_get_search_ranges()` level, though), but also throws a warning whenever this truncation happens, and instructs the user to manually set [minCoverFreq,maxCoverFreq] instead if there are problems. Because due to the Gaussian's infamous infinite tails and MCMC's small but nonvanishing chances to explore them, in principle no number of stds is sufficient to safely predict reaching outside the estimated coverage band.

Reply:
Changing the `MCMCSearch::test_fully_coherent` test case to a wide prior `"F0": {"type": "norm", "loc": F0, "scale": 1,` (and the Writer band from 4 to 10 tu support it), I can indeed produce the expected effect that a search can easily fail by trying to sample outside the `normal_stds=3`.

However I think there's no fundamental solution that can be easily provided here. The warning
```
WARNING  root:mcmc_based_searches.py:234 Gaussian priors (normal / half-normal) have been truncated at 3.000000 standard deviations for estimating the coverage frequency band. If sampling fails at any point, please consider manually setting [minCoverFreq,maxCoverFreq] to more generous values.
```
should be enough to guide the user to a workaround, even if it will require trial&error.

In most practical cases we'll use uniform, not normal, priors on frequency anyway, and this won't occur.

Reply:
What about being more conservative (like 5 sigmas)? 
I guess the answer is "Well, that will just make the error less likely ;)"
Anyways:
1. Follow Ups require well-defined "small" prior volumes to be effective. Why would you use a Gaussian then?
2. In case you need to use a Gaussian (what do you want to prove?), you'd be smart enough to locate it "properly" so the MCMC does not bail away....

I'm not sure of 2. being effective at all (the more I think about it, the less convinced I am), so let's go with the warning!

Reply:
Going to  more sigmas has the problem of in turn raising the chance of the provided SFTs not being wide enough. In some sense, having that error up front would be better than after a long sampling run randomly falls outside, but not sure it's worth it.